starter_code/CSS/style.css

```diff
@@ -0,0 +1,114 @@
+body{
+  font-family: 'Nunito Sans', sans-serif;
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  background-color: black;
+  height: 100px;
+  border-bottom: yellow solid 3px;
+}
+.logo {
+  color: yellow;
+  font-size: 40px;
+  padding: 10px;
+}
+nav {
+  /* justify-items: flex-end; */
+  align-self: flex-end;
+  padding: 10px;
+  display: flex;
+  justify-content: space-around;
+
+}
+
+nav a {
+  color: white;
+  font-size: 25px;
+  text-decoration: none;
+  padding: 10px;
+}
+.are {
+  text-decoration: yellow underline;
+}
+.taste {
+  text-decoration: line-through;
+  color: gray;
+  text-align: center;
+}
+main {
+  background-image: url('../images/coworking.jpg');
+   height: 700px;
+   background-repeat: no-repeat;
+   background-size: cover;
+}
+
+section {
+  height: 50px;
+  background-color: lightgray;
+  font-size: 30px;
+  text-align: center;
+  padding-top: 10px;
+}
+
+.create-profile{
+
+  float: right;
+  background-color:rgba(255,255,255,0.7);
+  border: 2px;
+  height:330px;
+  width: 250px;
+  margin-right: 25px;
+  padding: 15px;
+  text-align: center;
+  box-shadow:
+
+}
+.Meet{
+  font-weight: bold;
+}
+.create{
+  font-weight: bolder;
+  background-color: #FFFB00;
+  colop #000;
+  padding: 15px 35px;
+  border: none;
+  text-shadow: -1px 0px 0px rgba(0, 0, 0, 0.5);
+}
+.boxs{
+  display: grid;
+  grid-template-columns: 2fr 2fr 2fr ;
+  grid-gap:20px;
+
+
+
+}
+
+.box{
+  padding: 15px;
+  width: 300px;
```

Comment:
If you want the box content in the center, remove `width` & type `text-align: center;`